### PR TITLE
Add in-code documentation

### PR DESCRIPTION
**Pull request by niketagrawal**
- Add documentation to describe CLI implementation and how to add a new command.
- Fix typo
- Use the `name` argument consistently across the implementation of the commands in console.py

### CODE REVIEW

File: aeolis/console.py [hunk 1]
```diff
@@ -13,31 +14,88 @@
...
+
+bas.hoonhout@deltares.nl         b.m.hoonhout@tudelft.nl
+Deltares                         Delft University of Technology
+Unit of Hydraulic Engineering    Faculty of Civil Engineering and Geosciences
+Boussinesqweg 1                  Stevinweg 1
+2629 HVDelft                     2628CN Delft
+The Netherlands                  The Netherlands
+
+
+***************************************************************************
```
Comment (manuGil) [inline aeolis/console.py:27]: I discussed the IP issues with Sierd and he said that Bas was ok with doing whatever they wanted with the original source code. Sierd said he preferred to mention the AeoLiS team. if you discussed this with Sierd, please ignore these comments.

File: aeolis/console.py [hunk 2]
```diff
@@ -171,7 +230,7 @@ def start_aeolis_app():
 
 def start_aeolis_wind_app():
     """
-    This function serves the purpose of catching the deprecated command line syntx `aeolis-wind <path_to_wind.txt>` and pritning a message on the console instructing the user to use `aeolis wind <path_to_wind.txt>` instead.
+    This function serves the purpose of catching the deprecated command line syntax `aeolis-wind <path_to_wind.txt>` and pritning a message on the console instructing the user to use `aeolis wind <path_to_wind.txt>` instead.
```
Comment (manuGil) [inline aeolis/console.py:233]: Fix type in `pritning`, it should be  **printing**

**Review by manuGil — CHANGES_REQUESTED** [review]
@niketagrawal Please check my comments on your PR.

File: aeolis/console.py (hunk 1 above)
Comment (niketagrawal) [reply aeolis/console.py:27]: Thanks, I remember we discussed it already before. This is a typo on my end while trying to update the docstring. I will restore the old copyright text that we had which mentioned Aeolis development team.

File: aeolis/console.py (hunk 2 above)
Comment (niketagrawal) [reply aeolis/console.py]: Thanks for noticing this.

**Comment by niketagrawal** [comment]
Thanks @manuGil , comments are addressed.

**Review by manuGil — APPROVED** [review]
@niketagrawal It looks now. Thank for the corrections.

**Pull request merged**